plugins/prompt.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: prompt
+spec:
+  platforms:
+  - head: https://github.com/jordanwilson230/kubectl-plugins/archive/krew.zip
+    uri: https://github.com/jordanwilson230/kubectl-plugins/archive/v1.0.0-krew.zip
+    sha256: 21099ad3c3a6509a4ddb94aa0a88c055437e369c41c0e5c0cae49dc6db697241
+    bin: kubectl-prompt
+    files:
+    - from: "*/kubectl-prompt"
+      to: "."
+    selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+  version: "v1.0.0-krew"
```

Comment:
I recommend just doing this v1.0.0. we may start using semver soon to determine recentness (right now we rely on sha256).

Reply:
Since I'm cutting these releases off a branch of my kubectl-plugins repo, that might be a bit confusing to be releasing a v1.0.0 (vs v1.0.0-krew) that is specific only for Krew.  Does Krew recommend housing these contributions in a separate repo (i.e., outside jordanwilson230/kubectl-plugins)?

Reply:
> Since I'm cutting these releases off a branch of my kubectl-plugins repo, that might be a bit confusing to be releasing a v1.0.0 (vs v1.0.0-krew) that is specific only for Krew

I think it should be fine, given that the `platforms.uri` field still points to the actual `v1.0.0-krew` location.